dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -92,14 +92,47 @@
 
                 boolean isNumber = field.getFieldContentlet().startsWith(Field.DataType.INTEGER.toString());
                 boolean isFloat = field.getFieldContentlet().startsWith(Field.DataType.FLOAT.toString());
+                boolean isHostNameField = field.getVelocityVarName().equals("hostName");
+
+
+                String regex = (isNumber) ? "[0-9]*" : (isFloat) ? "[+-]?([0-9]*[.])?[0-9]+" : (isHostNameField) ? "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$" : "";
```

Comment:
Is this the same RegEx as the one in the `ESContentletAPIImpl.java` class? Maybe you can expose it and use it here as well.

Reply:
true. What do you think would be a good place to share that constant ? @jcastro-dotcms